lib/active_fedora/orders/association.rb

```diff
@@ -82,6 +82,12 @@ def delete_at(loc)
       target.delete_at(loc)
     end
 
+    # Delete all occurences of the specified target
+    # @param obj object to delete
+    def delete(obj)
```

Comment:
@jcoyne You should rename your new method to `#delete_target`. I get what you're trying to do, the problem is the relation you're adding this to controls `#ordered_member_proxies`. The argument to `#delete` should be an element that's returned from that relation, which is a proxy. Since you want to pass a member, it's deleting targets of proxies.